etc/kayobe/ansible/requirements.yml

```diff
@@ -3,7 +3,7 @@ collections:
   - name: stackhpc.cephadm
     version: 1.14.0
   - name: stackhpc.pulp
-    version: 0.4.1
+    version: 0.5.2
```

Comment:
Same for yoga? or is it another ansible/python version problem?

Reply:
I don't see anything incompatible. Might be good to keep yoga stable unless there's a specific fix we need though